.gitignore

```diff
@@ -115,4 +115,6 @@ model.py*
 /simulatedmodels.sh
 /.Rhistory
 */ExpressionsData.csv
-*.csv
\ No newline at end of file
+*.csv
+outputs/
```

Comment:
The name of the directory that users of BoolODE specify for their outputs can of course vary between users. I had to include the name of mine in the .gitignore file so that my outputs would not be pushed up. I do not know what to do about this besides just having this part of the merge being ignored.